Telerik.JustMock/Expectations/NonPublicRefReturnExpectation.cs

```diff
@@ -0,0 +1,194 @@
+﻿/*
+ JustMock Lite
+ Copyright © 2010-2018 Telerik EAD
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using Telerik.JustMock.Core;
+using Telerik.JustMock.Core.Context;
+
+namespace Telerik.JustMock.Expectations.Abstraction
+{
+    internal sealed class NonPublicRefReturnExpectation : NonPublicExpectationBase, INonPublicRefReturnExpectation
```

Comment:
I add this comment to clearly mark inheritance from NonPublicExpecatationBase which I find to be deviating from "IS A" OOP design principle.

Reply:
@lyubomir-rusev, fixed with [595ded1](https://github.com/telerik/JustMockLite/pull/42/commits/595ded10a032d8e33576643110280741ea96a82d)